lib/http.js

```diff
@@ -28,7 +29,7 @@ function httpRequest(sdk, options) {
     'Content-Type': 'application/json',
     'X-Okta-User-Agent-Extended': 'okta-auth-js-' + config.SDK_VERSION
   };
-  util.extend(headers, sdk.options.headers || {});
+  $.extend(headers, sdk.options.headers, options.headers);
```

Comment:
Let's make `util.extend` behave more like jQuery instead of importing jQuery.

Alternatively, you could call `util.extend` twice, but it'd be much cleaner if it accepted an arbitrary number of arguments.